src/components/Images.vue

```diff
@@ -379,4 +488,16 @@ export default {
   textarea#imageManagerOutput.finished {
     border: 2px solid dodgerblue;
```

Comment:
Huh, that should probably be based on the theme colours too.

Reply:
yup, I elaborated on it a bit to distinguish `--success` from `--failure`